finitespace/BME280
# Added reset() function

### Related issue # and issue behavior
I didn't create a related issue, because my problem has probably it's root cause not with this library, but is in fact an electronic problem. I have one of the BME280 chips in combination with an CCS881 connected to a NodeMCU-Board. While developing a working library for the CCS chip (the existing ones just don't work for me), the BME sometimes just kind of crashed and returned one or more values as "nan". Rebooting the ESP didn't help.
### Description of changes/fixes
To overcome this issue I looked up the documentation and found that the possibility of to reset the chip isn't currently implemented in this lib. So I took care of it ;)
### @mention a person to review
Maybe @finitespace wants to take a look? Of course, I appreciate every comment.
### Steps to test
1) Run BME until crash (yeah, that's.. not that easy if you don't have the problem.)
2) Call reset()
3) Call begin()
(Alternative: begin(), reset(), begin(), see if it's still running)
### Any outstanding TODOs or known issues
I don't know, maybe you prefer to put the necessary begin()-call into the reset()-function, I just implemented what the function name says it does. In my opinion that's a matter of taste.

Best regards,
Yannick
